akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -250,16 +250,21 @@ class CircuitBreaker(
    * Holds reference to current state of CircuitBreaker - *access only via helper methods*
    */
   @volatile
-  @silent("never used")
+  @nowarn("msg=never used")
```

Comment:
We still need this even though it _is_ now "used" below?

Reply:
This is the result of global find and replace.
I removed those two, but there are for sure other leftovers around ...

Reply:
Ok, doesn't hurt that much I guess.